examples/lasuscc/h4_sto3g.py

```diff
@@ -0,0 +1,64 @@
+# This is a sample script to run LAS-USCCSD for the H4 molecule with the polynomial-scaling algorithm
```

Comment:
It doesn’t look like that… It looks like it just computes the gradients but doesn’t select anything or run anything…